CHANGELOG.md

```diff
@@ -13,6 +13,9 @@
 [All changes in [[UnreleasedVersion]]](https://github.com/mozilla/uniffi-rs/compare/v0.12.0...HEAD).
 
 ### ⚠️ Breaking Changes ⚠️
+- uniffi no longer has ffi-support as a dependency.  This means it handles
+  panic logging on its own.  If you previously enabled the `log_panics` feature
+  for `ffi-support`, now you should enable it for `uniffi`.
```

Comment:
I think this is correct, but can someone double check that I'm saying the right thing here?

Reply:
That sounds correct to me. This patch is great - it took me some time to see how it worked before it clicked just how little `IntoFfi` was used.